product_docs/docs/biganimal/release/free_trial/index.mdx

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
@nataliawojcik27 @aswright491 , I think we can delete the Free trial topic in the full content. Everything is covered here. I don't see any value in having the content in two places in a slightly different format. Do you agree?

Reply:
per slack/zoom, Aislinn and I think leaving the content in two place is ok for now. thank you deedee!

Reply:
I added a link in the BAFT topic in the full version to the quick start topic. See https://github.com/EnterpriseDB/docs/pull/2354/commits/d9f934db875e1a56dabf8275f578330ae37fa436.